client/src/app/[tenant]/agents/[taskId]/[taskSchemaId]/playground/hooks/useIsProxy.tsx

```diff
@@ -0,0 +1,11 @@
+import { TaskSchemaResponseWithSchema } from '@/types/task';
+
+export function useIsProxy(schema: TaskSchemaResponseWithSchema) {
```

Comment:
for input variables you'll have to check if there is a `messages` key in the version properties